SampleApp/ViewController.swift

```diff
@@ -92,11 +92,14 @@ class ViewController: UIViewController, ARSCNViewDelegate {
             barChart = nil
         }
         
-        let values = generateRandomNumbers(withRange: 0..<50, numberOfRows: 50, numberOfColumns: 50)
-            
+//        let values = generateRandomNumbers(withRange: 0..<50, numberOfRows: 50, numberOfColumns: 50)
+        let values = generateNumbers(fromDataSampleWithIndex: 1)!
+        
         let dataSeries = ARDataSeries(withValues: values)
         self.barChart = ARBarChart(dataSource: dataSeries, delegate: dataSeries, size: SCNVector3(0.2, 0.2, 0.2))
         self.barChart.position = position
+        self.barChart.animationType = .grow
+        self.barChart.drawGraph()
```

Comment:
That was my struggle for the longest time yesterday =\
It doesn't seem like it. I couldn't find any lifecycle functions to overwrite for `SCNNode`.